src/nxdoc/nuxeo-server/installation/how-to-estimate-volume-usage.md

```diff
@@ -177,11 +177,15 @@ The database will store:
 *   Extracted text volume x2
 *   Metadata
 
+Note that the volume depends a lot of the backend, some database will not compress data
+other will compress big fields (MongoDB will compress everything).
```

Comment:
Note that the volume depends a lot **on** the backend: some databases will not compress data, other will compress big fields and MongoDB will compress everything.